OpenEdXMobile/res/layout/section_row_layout.xml

```diff
@@ -13,7 +13,9 @@
     <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:padding="@dimen/edx_default_margin">
+        android:layout_margin="@dimen/edx_divider_length"
```

Comment:
Still getting 1dp space in the expended view.

Reply:
i knew that because of the dynamic borders, the issue could either be encountered in the expanded state or in the collapsed state. One solution was causing problems for the other. So, I decided to leave it as it is for the expanded state.

Reply:
Please handle the margin added in subSectionLayout, it will fix this